cmd/syncthing/main.go

```diff
@@ -201,6 +203,9 @@ are mostly useful for developers. Use with care.
                    "h", "m" and "s" abbreviations for hours minutes and seconds.
                    Valid values are like "720h", "30s", etc.
 
+ STNOTRAY          Prevents syncthing from hiding the console and starting up
+                   with a tray icon. (Windows only)
```

Comment:
Does this mean the default is to hide console? For double clicking an exe that seems fine, for someone running it from cmd this seems annoying. I am just asking because my current windows understanding is power user == syncthing exe and normal user == synctrayzor. So I am wondering whether such a default (if it is the case) is a disservice to power user. Just a thought, I am no windows user.

Reply:
To be honest, I am thinking if we even need this. This is to force launch without a tray from explorer, but that means that you have to set the env var first, which probably means setting it OS wide.